microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -317,6 +296,74 @@ To read more about DOM leaks and memory analysis fundamentals, check out [Findin
 <!--[narrow down causes of memory leaks](../profile/memory-problems/memory-diagnosis#narrow-down-causes-of-memory-leaks) -->
 
 
+<!-- ====================================================================== -->
+## Analyze the impact of closures on memory
+
+To analyze the impact of closures on memory, try out this example:
+
+1. Open the [Eval is evil](https://microsoftedge.github.io/Demos/devtools-memory-heap-snapshot/example-07.html) demo webpage in a new window or tab.
+
+1. Record a first heap snapshot.
```

Comment:
```suggestion
1. Record a heap snapshot.
```
reword simpler, to avoid my mis-parsing that happened:
Record a snapshot of type "first heap".
Record a first-heap snapshot.  (What's a first-heap snapshot?)